general/management/commands/create_front_page_caches.py

```diff
@@ -33,10 +35,15 @@ def handle(self, **options):
         logger.info("Updating front page caches")
 
         rss_url = settings.FREESOUND_RSS
-        pledgie_campaign = settings.PLEDGIE_CAMPAIGN
-        
+        donations_goal = settings.DONATIONS_GOAL
+        donations_date = settings.DONATIONS_DATE
+
         rss_cache = render_to_string('rss_cache.html', locals())
         cache.set("rss_cache", rss_cache, 2592000) # 30 days cache
 
-        pledgie_cache = render_to_string('pledgie_cache.html', locals())
-        cache.set("pledgie_cache", pledgie_cache, 2592000) # 30 days cache
\ No newline at end of file
+        donations = accounts.models.Donation.objects\
+                .filter(created__gt=donations_date).all().aggregate(Sum('amount'))
```

Comment:
My first implementation was using a model with campaigns, which I changed for this simplified version. I will change it now.
About currencies, in the donations form I specify the currency I want to receive the donation (I think you also can set this in the configuration of the account). So if the account of the donor is in other currency paypal makes a conversion.